### ISSUE TITLE
Can RND be used in face blind restoration

### ISSUE THREAD

**Issue by helloful** [wyhuai/RND#1]
This is excellent work and I would like to ask if RND can be used in face blind restoration tasks

**Comment by wyhuai** [comment]
Yes! Our further work applies RND to diffusion models to implement diverse **zero-shot** image restoration tasks, including blind restoration tasks! The source code will be released soon!

**Comment by wyhuai** [comment]
> This is excellent work and I would like to ask if RND can be used in face blind restoration tasks

Yes! Our further work applies RND to diffusion models to implement diverse zero-shot image restoration tasks, including blind restoration tasks! The source code will be released soon!

**Comment by wyhuai** [comment]
> This is excellent work and I would like to ask if RND can be used in face blind restoration tasks

The code is here, https://github.com/wyhuai/DDNM

**Comment by helloful** [comment]
> > This is excellent work and I would like to ask if RND can be used in face blind restoration tasks
> 
> The code is here, https://github.com/wyhuai/DDNM

Thank you very much for your reply, and I hope your paper will be accepted.
